Models/Dto/Bookmark.cs

```diff
@@ -0,0 +1,16 @@
+using System.ComponentModel.DataAnnotations;
+using System;
+
+namespace Dto
+{
+    public class Bookmark
+    {
+        public int Id { get; set; }
+        public string Link { get; set; } = string.Empty;
```

Comment:
The Link should only be in your DTO not saved into your DB since it might change from version to version.
You should calculate the Link when you create the DTO object.